.gitignore

```diff
@@ -25,3 +25,7 @@ bundle/
 .vagrant/
 phantomjsdriver.log
 selenium-debug.log
+
+*~
```

Comment:
Can you put this in a per-user ignore rather than the project-wide ignore? I personally really like git to tell me when random tools or editors have decided to leave trash lying around my directory tree.